src/main/java/fr/jmmc/oiexplorer/core/gui/SliderPanel.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package fr.jmmc.oiexplorer.core.gui;
+
+import fr.jmmc.oitools.image.FitsImage;
+import java.util.List;
+
+/**
+ *
+ * @author martin
+ */
+public final class SliderPanel extends javax.swing.JPanel {
+
+    List<FitsImage> fitsImages;
+    FitsImagePanel fitsImagePanel;
+    SliderEditor sliderEditor;
+    
+    public SliderPanel() {
+        initComponents();
+    }
+    /**
+     * Creates new form SliderPanel
+     * @param fitsImagePanel
+     */
+    public SliderPanel(FitsImagePanel fitsImagePanel) {
+        initComponents();
+        
+        this.fitsImagePanel = fitsImagePanel;
+    }
+
+    /**
+     * This method is called from within the constructor to initialize the form.
+     * WARNING: Do NOT modify this code. The content of this method is always
+     * regenerated by the Form Editor.
+     */
+    @SuppressWarnings("unchecked")
+    // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
+    private void initComponents() {
+
+        jFitsImageCubeSlider = new javax.swing.JSlider();
+        jSliderButton = new javax.swing.JButton();
+
+        setLayout(new java.awt.GridBagLayout());
+
+        jFitsImageCubeSlider.addChangeListener(new javax.swing.event.ChangeListener() {
+            public void stateChanged(javax.swing.event.ChangeEvent evt) {
+                jFitsImageCubeSliderStateChanged(evt);
+            }
+        });
+        add(jFitsImageCubeSlider, new java.awt.GridBagConstraints());
+        this.jFitsImageCubeSlider.setValue(1);
+        this.jFitsImageCubeSlider.setMinimum(1);
+        this.jFitsImageCubeSlider.setMaximum(1);
+
+        jSliderButton.setText("Edit range");
+        jSliderButton.addMouseListener(new java.awt.event.MouseAdapter() {
+            public void mouseClicked(java.awt.event.MouseEvent evt) {
+                jSliderButtonMouseClicked(evt);
+            }
+        });
+        add(jSliderButton, new java.awt.GridBagConstraints());
+    }// </editor-fold>//GEN-END:initComponents
+
+    private void jFitsImageCubeSliderStateChanged(javax.swing.event.ChangeEvent evt) {//GEN-FIRST:event_jFitsImageCubeSliderStateChanged
+        if (this.fitsImagePanel != null && this.fitsImages != null && this.fitsImages.size() > 1) {
```

Comment:
I wonder how to generalize such action using Listeners ...
FYI In Aspro2 I made UserModelAnimatorListener to have a single animator (like slider) but several components registered to display images in sync.
Anyway I will revisit that later